Description: Adjust filter costs so application/vnd.adobe-read-postscript input doesn't go via pstotiff 
Bug-Redhat: https://bugzilla.redhat.com/show_bug.cgi?id=1008166
Author: Tim Waugh <twaugh@redhat.com>
Last-Update: 2013-11-28
diff -up cups-filters-1.0.41/mime/cupsfilters.convs.filter-costs cups-filters-1.0.41/mime/cupsfilters.convs
--- cups-filters-1.0.41/mime/cupsfilters.convs.filter-costs	2013-11-19 11:51:06.279294346 +0000
+++ cups-filters-1.0.41/mime/cupsfilters.convs	2013-11-19 11:51:07.960301722 +0000
@@ -85,7 +85,7 @@ application/vnd.cups-pdf	application/vnd
 #
 
 application/vnd.cups-pdf	application/vnd.cups-raster	99	gstoraster
-application/vnd.cups-postscript	application/vnd.cups-raster	200	gstoraster
+application/vnd.cups-postscript	application/vnd.cups-raster	175	gstoraster
 application/vnd.cups-pdf	application/vnd.cups-raster	100	pdftoraster
 image/gif			application/vnd.cups-raster	100	imagetoraster
 image/png			application/vnd.cups-raster	100	imagetoraster
